src/main/webapp/encounters/encounter.jsp

```diff
@@ -879,7 +879,7 @@ else {
 
 
 <%
-if(isOwner && enc.getLocation()!=null){
+if((isOwner || isPublic) && enc.getLocation()!=null){
 %>
```

Comment:
this clause, since it is only _displaying_ a value, should be `(isOwner || isPublicViewable)`.